src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,27 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
+  // return standard img url pips
```

Comment:
I don't really understand this comment

Reply:
Changed the comment to a more specific one.

Reply:
I also changed a test descriptionto mirror the change.
This comment was previously based on that test description